site/docs/cli/run.md

```diff
@@ -12,6 +12,10 @@ This command runs the Envoy AI Gateway locally as a standalone proxy with a give
 Since the project is primarily focused on the Kubernetes environment, this command is useful for testing the configuration locally before deploying it to a Kubernetes cluster.
 Not only does it help in testing the configuration, but it is also useful in a local development environment of the provider-agnostic AI applications.
 
+:::warning
+Currently, `aigw run` only properly works on Linux. macOS support is pending due to the Envoy v1.34+ binary not available for macOS yet. That will be resolved with https://github.com/Homebrew/homebrew-core/pull/223051.
```

Comment:
technically this is use at your own risk, right? as metrics won't work until that? or there is something bigger. The workaround is to use docker when on macOS because aigw doesn't use docker internally. That said, I didn't see a guide or doc saying where the docker images are published to. Might be nice to put that somewhere. https://aigateway.envoyproxy.io/docs/cli/aigwinstall

Reply:
yeah technically at your own risk but i guess there's something bigger like https://github.com/envoyproxy/ai-gateway/issues/650

Meanwhile, as we haven't published the docker image yet, it makes sense to do so as a workaround